pyndl/types.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, Iterator, Tuple, TypeVar
+
+
+from numpy import ndarray
+from xarray.core.dataarray import DataArray
```

Comment:
See @derNarr's comments above about how to declare types

Reply:
I don't think his suggestion can be applied here: 
In the other files, numpy / xarray is already importet.